Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentImportMenu.swift

```diff
@@ -71,6 +71,7 @@ struct AttachmentImportMenu: View {
         }
     }
     
+    @available(visionOS, unavailable)
     private func takePhotoOrVideoButton() -> Button<some View> {
```

Comment:
We don't have camera access on the Vision Pro so we shouldn't present this option. If the user tries to use the camera, the device will use your personal persona as the camera feed. So it seems best to not allow this option.